atomics/include/desul/atomics/Lock_Array_HIP.hpp

```diff
@@ -23,6 +23,8 @@ namespace Impl {
 
 #ifdef __HIP_DEVICE_COMPILE__
 #define DESUL_IMPL_BALLOT_MASK(x) __ballot(x)
+#else
+#define DESUL_IMPL_BALLOT_MASK(x) 0
```

Comment:
Please elaborate about that change.

Reply:
The macro needs to exist on the both the host and the device but you cannot call `__ballot` on the host